apps/core/scenes/results/__generated__/ResultsListItem.graphql.js

```diff
@@ -11,60 +11,150 @@ import type { ConcreteFragment } from 'relay-runtime';
 import type { FragmentReference } from "relay-runtime";
 declare export opaque type ResultsListItem$ref: FragmentReference;
 export type ResultsListItem = {|
-  +price: ?number,
-  +flyFrom: ?string,
-  +flyTo: ?string,
-  +localDeparture: ?string,
-  +localArrival: ?string,
+  +price: ?{|
+    +currency: ?string,
+    +amount: ?number,
+  |},
+  +route: ?$ReadOnlyArray<?{|
+    +airline: ?string,
+    +arrival: ?{|
+      +city: ?string,
+      +cityCode: ?string,
+      +localTime: ?any,
+      +utcTime: ?any,
+    |},
+    +departure: ?{|
+      +city: ?string,
+      +cityCode: ?string,
+      +localTime: ?any,
+      +utcTime: ?any,
```

Comment:
these generated flow's types look useless. It looks like I can expect whatever back from graphql